packages/components/src/components/hint-box/hint-box.scss

```diff
@@ -7,6 +7,25 @@
     padding: 8px;
     border-radius: $BORDER_RADIUS;
 
+    &--inline {
+        margin-top: 0.8rem;
+        padding: 0.8rem;
+        display: flex;
+        align-items: flex-start;
+        background: rgba(255, 173, 58, 0.16);
```

Comment:
nit: The same gradient is being used in 2 other places in the app. May be we can move this to themes?

Reply:
Just notice that the background is not required since there's `background-color` which already included inside `--info`. Will remove this line of code